### ISSUE TITLE
ref_storage

### ISSUE THREAD

**Issue by rb004f** [lanl-ansi/GasModels.jl#234]
in ref_storage, we have an equivalence between pressure and density, e.g., storage["reservoir_density_max"] = storage["reservoir_p_max"].  Shouldn't there be a conversion factor here?

**Comment by kaarthiksundar** [comment]
This is correct from the perspective of transient. The non-dimensionalization for all transient formulations is performed so that the non-dimensional pressure == non-dimensional density. If we use a different non-dimensionalization (say if we start with flow scaling, like some instances do), then this is incorrect. Will have to see how this can be fixed.

**Comment by rb004f** [comment]
This will get addressed in the unification of the transient and steady-state specifications
